docs/sphinx/examples/1d_stencil.rst

```diff
@@ -82,7 +82,7 @@ string dataflows together and construct an execution tree.
 
 After the values of the futures in dataflow are ready, the values must be pulled
 out of the future container to be passed to the function ``heat``. In order to
-do this, we use the HPX facility ``unwrapped``, which underneath calls
+do this, we use the HPX facility ``unwrapping``, which underneath calls
```

Comment:
The facility is actually called `unwrapped`, so this is not a typo.

Reply:
I renamed `unwrapped` back to `unwrapping`.